packages/@uppy/core/src/index.js

```diff
@@ -49,6 +49,7 @@ class Uppy {
         youCanOnlyUploadFileTypes: 'You can only upload: %{types}',
         companionError: 'Connection with Companion failed',
         companionAuthError: 'Authorization required',
+        companionUnauthorizeHint: 'To unauthorize to your %{provider} account, please go to %{url}',
```

Comment:
This is not super useful probably since the link is not clickable and users won't be able to copy it before the bubble disappears. Maybe we could make it say something like "You can now remove the authorization in %{provider}'s settings panel"? Would be nice to actually link to the page but that would require some changes in the info bubbles...

Reply:
We could make the link clickable (by setting content of the informer with `__dangerouslySetInnerHTML`) and set the bubble to hold for 10 seconds? Until we add support for alert modals with “ok” (or we could use actual alert? I think it would be too intrusive and ugly in this case).

Reply:
my thought is that the URL being displayed now at least helps users find the URL where to revoke this access. Otherwise, some of these settings can be hard to find. Is there someway to make informer stay longer maybe?

Reply:
3rd argument to `uppy.info()` is duration in milliseconds

Reply:
thank you! I have extended the duration of the message to last for 7 seconds, and it seemed long enough for me to highlight and copy the message.